spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/support/H2PagingQueryProvider.java

```diff
@@ -22,14 +22,15 @@
  * H2 implementation of a {@link PagingQueryProvider} using database specific features.
  *
  * @author Glenn Renfro
+ * @author Chris Bono
  */
 public class H2PagingQueryProvider extends AbstractSqlPagingQueryProvider {
 
 	@Override
 	public String getPageQuery(Pageable pageable) {
-		String topClause = new StringBuilder().append("LIMIT ").append(pageable.getOffset()).append(" ")
-				.append(pageable.getPageSize()).toString();
-		return SqlPagingQueryUtils.generateTopJumpToQuery(this, topClause);
+		String limitClause = new StringBuilder().append("OFFSET ")
```

Comment:
**[In general, H2 has gotten stricter about what they allow and the syntax/parsing]**

This is the replacement for the previous `LIMIT M N`.